### PR TITLE
Dnn 8585: databaseOwner issue 

### PR DESCRIPTION
**Pull request by sleupold**
fix

### CODE REVIEW

File: Website/Providers/DataProviders/SqlDataProvider/08.00.00.13.SqlDataProvider [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-IF EXISTS (SELECT * FROM {databaseOwner}sysobjects WHERE id = object_id(N'{databaseOwner}[{objectQualifier}AddException]') AND OBJECTPROPERTY(id, N'IsPROCEDURE') = 1)
+IF EXISTS (SELECT * FROM sys.procedures WHERE id = object_id(N'{databaseOwner}[{objectQualifier}AddException]'))
```
Comment (amarjit-dhunna) [inline Website/Providers/DataProviders/SqlDataProvider/08.00.00.13.SqlDataProvider]: Please change the Where id= to Where object_id

**Comment by sleupold** [comment]
done

**Comment by jbrinkman** [comment]
This was resolved with a different commit.  We may still want to create a corrected update to ensure any "broken" installs are fixed.

**Comment by zyhfish** [comment]
Already resolved in #c2b0979d.

**Pull request closed without merging**